src/components/frame/v5/pages/FundsPage/partials/Filter/Filter.tsx

```diff
@@ -0,0 +1,315 @@
+                    <NestedFilterItem
+                      onChange={onChange}
+                      path={`${path}.${nestedItem.name}`}
+                      value={value}
+                      {...nestedItem}
+                    />
+                  ))}
+                </motion.div>
+              )}
+            </AnimatePresence>
+          )}
+        </>
+      ) : (
+        <>
+          <div ref={setNestedTriggerRef}>
+            <Checkbox
+              isChecked={isChecked}
+              onChange={(e) => {
+                onChange(set(cloneDeep(value), path, e.target.checked));
+              }}
+              classNames="subnav-button px-0 sm:px-3.5"
+            >
+              {label}
+            </Checkbox>
+          </div>
+          {!!nestedItems?.length && isChecked && (
+            <PopoverBase
+              setTooltipRef={setNestedTooltipRef}
+              tooltipProps={getNestedTooltipProps}
+              withTooltipStyles={false}
+              cardProps={{
+                rounded: 's',
+                hasShadow: true,
+                className: 'py-6 px-2',
+              }}
+              classNames="w-full sm:max-w-[13.25rem] mr-2"
+            >
+              {nestedItems.map(({ ...nestedItem }) => (
+                <NestedFilterItem
+                  onChange={onChange}
+                  path={`${path}.${nestedItem.name}`}
+                  value={value}
+                  {...nestedItem}
+                />
+              ))}
+            </PopoverBase>
+          )}
+        </>
+      )}
+    </>
+  );
+}
+
+function RootFilter<TValue extends FilterValue>({
+  items,
+  label,
+  iconName,
+  onChange,
+  path,
+  value,
+  title,
+}: RootFilterProps<TValue>) {
+  const { getTooltipProps, setTooltipRef, setTriggerRef, visible } =
+    usePopperTooltip({
+      delayShow: 200,
+      delayHide: 200,
+      placement: 'left-start',
+      interactive: true,
+    });
+  const isMobile = useMobile();
+  const [isAccordionOpen, { toggle: toggleAccordion }] = useToggle();
+
+  return (
+    <>
+      {isMobile ? (
+        <AccordionItem
+          isOpen={isAccordionOpen}
+          onToggle={toggleAccordion}
+          title={title}
+          iconName="chevron-down"
+          className="[&_.accordion-toggler]:text-gray-400 [&_.accordion-toggler]:text-4 [&_.accordion-toggler]:uppercase sm:[&_.accordion-toggler]:px-3.5 mb-4 last:mb-0"
+        >
+          {items.map((item) => (
+            <NestedFilterItem<TValue, 2>
+              label={item.label}
+              name={item.name}
+              items={item.items}
+              onChange={onChange}
+              value={value}
+              path={`${path}.${item.name.toString()}`}
+            />
+          ))}
+        </AccordionItem>
+      ) : (
+        <>
+          <button
+            type="button"
+            className="subnav-button gap-3 px-0 sm:px-3.5"
+            ref={setTriggerRef}
```

Comment:
missing aria label

Reply:
This button have texts inside, I don't think aria-label is needed